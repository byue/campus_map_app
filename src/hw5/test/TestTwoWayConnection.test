# Tests that a graph can be created with 2 nodes
# that are connected in both directions.

# Create a graph
CreateGraph graph1

# Add 2 nodes
AddNode graph1 n1
AddNode graph1 n2

# Connect nodes in both directions
AddEdge graph1 n1 n2 a
AddEdge graph1 n2 n1 b

# list nodes of graph
ListNodes graph1

# list children of each node
ListChildren graph1 n1
ListChildren graph1 n2