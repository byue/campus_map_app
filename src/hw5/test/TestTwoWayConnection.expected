# Tests that a graph can be created with 2 nodes
# that are connected in both directions.

# Create a graph
created graph graph1

# Add 2 nodes
added node n1 to graph1
added node n2 to graph1

# Connect nodes in both directions
added edge a from n1 to n2 in graph1
added edge b from n2 to n1 in graph1

# list nodes of graph
graph1 contains: n1 n2

# list children of each node
the children of n1 in graph1 are: n2(a)
the children of n2 in graph1 are: n1(b)